tests/utils.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { shallow as enzymeShallow, mount as enzymeMount } from 'enzyme'
+import getMuiTheme from 'material-ui/styles/getMuiTheme'
+import createStore from 'store/createStore'
+import Styletron from 'styletron-client'
+
```

Comment:
I don't think we need these just yet. You certainly don't need all that context for shallowMount on functional components